index.js

```diff
@@ -383,9 +395,9 @@ module.exports = {
     if (_.isUndefined(dsEntry)) {
       return done(
         new Error(
-          `Consistency violation: Cannot do that with datastore (${datastoreName}) 
```

Comment:
Hey,
Can we remove this whitespace?

Reply:
Already done

Reply:
Corrected those format changes. Had to do a force push.